spec/models/reservation_spec.rb

```diff
@@ -494,6 +494,46 @@
     end
   end
 
+  context 'when manually over-extending reservation' do
+    before(:each) do
+      @em = FactoryGirl.create(:equipment_model)
+      @ei = FactoryGirl.create(:equipment_item, equipment_model: @em)
+
+      reservation.update_attributes(equipment_item: @ei,
+                                    equipment_model: @em,
+                                    start_date:
+                                    Time.zone.today + 1.day,
+                                    due_date:
+                                    Time.zone.today + 2.days)
+
+      @r2 = FactoryGirl.build(:valid_reservation,
+                              equipment_item: @ei,
+                              equipment_model: @em,
+                              start_date: Time.zone.today +
+                                          3.days,
+                              due_date: Time.zone.today +
+                                        4.days)
+    end
+
+    it 'should save reservation' do
+      expect(reservation.save).to be_truthy
+    end
+
+    it 'should have equipment model with 1 item' do
+      expect(@em.equipment_items.count).to eq(1)
+    end
+
+    it 'should save before over-extending' do
```

Comment:
If we remove this, we can change [L509](https://github.com/YaleSTC/reservations/pull/1256/files#diff-34f9cc630e9d84f6ade076d70a765b70R509) to `FactoryGirl.create` and skip the `@r2.save` call on [L531](https://github.com/YaleSTC/reservations/pull/1256/files#diff-34f9cc630e9d84f6ade076d70a765b70R531) below.